src/widget/wtracktableviewheader.cpp

```diff
@@ -42,7 +42,11 @@ HeaderViewState::HeaderViewState(const QString& base64serialized) {
     array.append(base64serialized);
     // First decode the array from Base64, then initialize the protobuf from it.
     array = QByteArray::fromBase64(array);
-    m_view_state.ParseFromArray(array.constData(), array.size());
+    if (!m_view_state.ParseFromArray(array.constData(), array.size())) {
+      qDebug() << "ERROR: Could not parse m_view_state from QByteArray of size "
```

Comment:
Please check for 4 space indent in this PR and 8 space or other non 4 Space indent for line breaks 
"qDebug()::operator<<" connects the strings and adds a space for us.